### PR TITLE
python3.pkgs.duckdb-engine: 0.13.2 -> 0.13.4

### PR DESCRIPTION
**Pull request by cpcloud**
Bump duckdb-engine to the latest release.

`jupysql` (which has duckdb-engine as dependency) will continue to fail, and is failing with the existing duckdb-engine release (0.13.2).

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by tobim** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 352852`

---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python311Packages.jupysql</li>
    <li>python311Packages.jupysql.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python311Packages.duckdb-engine</li>
    <li>python311Packages.duckdb-engine.dist</li>
    <li>python312Packages.duckdb-engine</li>
    <li>python312Packages.duckdb-engine.dist</li>
  </ul>
</details>

**Comment by tobim** [comment]
How does this relate to https://github.com/NixOS/nixpkgs/pull/352864?

**Comment by cpcloud** [comment]
> How does this relate to #352864?

I tried to indicate that in the PR description:

> jupysql (which has duckdb-engine as dependency) will continue to fail, and is failing with the existing duckdb-engine release (0.13.2).

Originally I thought this might fix the issue, but it doesn't. I did the upgrade anyway, so I thought'd I just push it up for review.

**Comment by sarahec** [comment]
Result of `nixpkgs-review pr 352852` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages failed to build:</summary>
  <ul>
    <li>python311Packages.jupysql</li>
    <li>python311Packages.jupysql.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql.dist</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python311Packages.duckdb-engine</li>
    <li>python311Packages.duckdb-engine.dist</li>
    <li>python312Packages.duckdb-engine</li>
    <li>python312Packages.duckdb-engine.dist</li>
  </ul>
</details>

**Comment by cpcloud** [comment]
Can this be merged?

**Review by sarahec — APPROVED** [review]
LGTM

**Comment by sarahec** [comment]
> Can this be merged?

I expect this will be picked up quickly now. (The approval delay was waiting for ofborg to complete its checks)

**Comment by cpcloud** [comment]
@sarahec Any chance you'd be willing to merge this?

**Comment by sarahec** [comment]
I've put in a request for the committers. They normally catch packages as they're approved, unless they're a few days back like this one.

**Pull request merged**
